FordLabs/PeopleMover
# Create matomo events

## Issue
Resolves #481 

## What was done
Add matomo events with space name in the category for:
- [x] Create Product
- [x] Delete Product
- [x] Edit Product
- [x] Add Person
- [x] Assign Person
- [x] Cancel Assignment
- [x]  Mark as Placeholder
- [x]  UnMark as Placeholder
- [x] Share Access

## How to test
1. Use these features in a space
2. Verify the events are showing up with space name in the category on Matomo

Feature URL: https://featurebranchui.apps.pd01i.edc1.cf.ford.com/
Feature Branch Deployment: https://jenkins-fordlabs.apps.pd01e.edc1.cf.ford.com/job/PeopleMover/job/Deploy%20PeopleMover%20FeatureBranch/
